content/en/quality_gates/_index.md

```diff
@@ -22,35 +25,92 @@ further_reading:
 <div class="alert alert-warning">Quality Gates is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
 {{< /site-region >}}
 
-{{< callout url="#" btn_hidden="true" >}}
-Quality Gates is in public beta.
-{{< /callout >}}
-
 ## Overview
 
-Quality Gates allows you to gate your workflows based on signals in Datadog. 
+Quality Gates allow you to control software quality by configuring rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.
+
+{{< img src="quality_gates/setup/pipeline_rule.png" alt="A pipeline rule that fails when code coverage for PCT is below or equal to zero in Quality Gates" style="width:100%" >}}
+
 Use Quality Gates to:
 
-* Create rules that can block your workflows by using data in Datadog.
-* Give your teams control over what code makes it to production and what doesn't.
-* Continually improve code quality with precise enforcement.
+* Create rules that block workflows using data in Datadog, ensuring that only code that meets your standards end up in production.
+* Give your organization the ability to decide what code makes it to production, enhancing your deployment discipline and mitigating potential issues in live environments.
+* Continually improve code quality and system performance with precise enforcement and customizable rules.
+
+By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives. 
```

Comment:
```suggestion
By integrating Quality Gates [into your CI/CD pipelines][7], you can create a robust framework for maintaining and improving software quality that aligns with your organization's operational goals and business objectives. 
```
This link takes me to /monitors/guide/github_gating/ -> is this correct ?

Reply:
Yes, this was originally a guide in the CI Visibility doc set that we migrated to the Monitors doc set!